### DISCUSSION TITLE
Welcome to RTPS Discussions!

### DISCUSSION THREAD

**Discussion by danieleades** [danieleades/rtps#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kczimm** [comment]
Hey, my name is Kevin. I'm interested in DDS for Rust and am hoping to help contribute to a design that has an async API. I recognize it starts with RTPS and so I want to help make a Rust RTPS implementation that fulfills the goals of this repo i.e. is idiomatic Rust and stays faithful to the specification.

**Comment by danieleades** [comment]
My name is Daniel. I work for a robotics software company interested in DDS and ROS2. In my spare time I write a lot of Rust.
I'm not actively using Rust at work, but a rock-solid DDS implementation would go a long way to making that possible.

I'm primarily interested in contributing to the existing [RustDDS](https://github.com/jhelovuo/RustDDS). I am using *this* crate as a proving ground for exploring design patterns that could be used to improve or extend RustDDS.

That said, I'm also open to this crate becoming a solid RTPS framework/implementation in its own right. Let's see what happens.
